### ISSUE TITLE
Support for Label in NumericUpDown?

### ISSUE THREAD

**Issue by SKProCH** [AvaloniaCommunity/Material.Avalonia#301]
### Discussed in https://github.com/AvaloniaCommunity/Material.Avalonia/discussions/299

<div type='discussions-op-text'>

<sup>Originally posted by **seikosantana** October  2, 2023</sup>
Hi, I tried using `assists:TextFieldAssist.Label` on a `NumericUpDown` but it doesnt seem to make any labels like it would on a TextBox. Is there a way to get the label just like in TextBox so it would look consistent?</div>

**Comment by GlitchyPSIX** [comment]
mm... this doesn't seem to have moved in 2 years?

**Comment by SKProCH** [comment]
Yes, this should be in scope of #327, but i totally miss free time to do complete it.
